lib/tasks_private/oracle_cloud.rake

```diff
@@ -0,0 +1,20 @@
+namespace :oracle_cloud do
+  namespace :regions do
+    desc "Update list of regions"
+    task :update => :environment do
+      require "oci"
+
+      regions = OCI::Regions::REGION_ENUM.map do |name|
```

Comment:
Is this hardcoded into the gem?  If so, we should not generate the yaml at all and instead override the base method to just to this directly